client/context/context.go

```diff
@@ -29,6 +29,7 @@ type CLIContext struct {
 	Height          int64
 	NodeURI         string
 	FromAddressName string
+	Signer          string
```

Comment:
Is any docs stating that how many contexts we defined and what 's their use case ?